### DISCUSSION TITLE
Replacing itext on BIRT version 4.8

### DISCUSSION THREAD

**Discussion by cedric-seq** [eclipse-birt/birt#1740] [General]
Hi,
I wanted to know if anyone that can help us replace the itext version in the BIRT 4.8. Are they any other approaches that you could recommend ? We found that upgrading to latest may not be an option when considering that there are a large number of rpts that may need to be upgraded. 

Thanks

**Comment by hvbtup** [comment]
Why do you think that reports may need to be upgraded?
